actiontext/lib/action_text/attribute.rb

```diff
@@ -46,6 +46,11 @@ def #{name}=(body)
         CODE
 
         rich_text_class_name = encrypted ? "ActionText::EncryptedRichText" : "ActionText::RichText"
+
+        rich_text_class_name.safe_constantize.class_eval do
+          has_many_attached :embeds, service: attachment_service
```

Comment:
Would not this add many different associations to `embeds` with different service if you call this method more than once? I don't think that is what we want.